CRAMv3.tex

```diff
@@ -2622,6 +2622,7 @@ \subsubsection*{rANS order-1}
 
 \Statex
 \Procedure{RansDecode1}{$output$, $nbytes$}
+  \State \Call{ReadFrequencies1}{$F,\ C$} \Comment{Creates F and C vectors}
```

Comment:
Might be worth updating the comment to reflect the fact that `F` and `C` are (2d) arrays in this context.